---
description: >-
  The OSDR data repository contains lots of information and data from the Space
  Biology research community.
---

# Access data in the OSDR

1. Go to the **NASA OSDR** [home page](https://osdr.nasa.gov/bio/index.html).
2. Click on the **"Data & Tools"** tab at the top of the page

_or click on the_ [_Data Repository Explo_](https://osdr.nasa.gov/bio/repo/search?q=\&data\_source=cgene,alsda\&data\_type=study)_re Image_&#x20;

<figure><img src=".gitbook/assets/Slide3 (2).png" alt="image showing what the button for the Data Repository Explorer looks like" width="375"><figcaption><p><a href="https://osdr.nasa.gov/bio/repo/search?q=&#x26;data_source=cgene,alsda&#x26;data_type=study"><em>or click on the Data Repository Explore Image</em></a> </p></figcaption></figure>

1. Select [**“Data Repository”**](https://osdr.nasa.gov/bio/repo/search?q=\&data\_source=cgene,alsda\&data\_type=study) to navigate to the list of studies ([Or use this link](https://osdr.nasa.gov/bio/repo/search?q=\&data\_source=cgene,alsda\&data\_type=study)).
2. Search for studies by entering text into the **“Search Datasets”** box.
3. On the right you’ll see the number of items per page, the default is 25, but this number can be increased to 50 or 100.
4. These lists are sorted based on **“Release Date”** as default, but can be adjusted to reflect **“Relevance”,** **“study accession ID”** # or the **“Study title”**.
5. On the left of the webpage, you will see some filters to help you identify related studies, select any combination of study features by ‘checking the boxes’ and this will change the number of studies being shown on the screen.
6. Each study is summarized by an image representing the model organism being investigated, the study title, organism, factors being investigated, assay types, release date, a description and a “Highlights”/ summary.
7. Clicking on either the image of the model organism or the title will open that study database and present the related metadata, primary data and if available processed data.

<figure><img src=".gitbook/assets/Slide5 (1).png" alt="The Open Science Data Repository search interface allows users to search, filter, and view search results, including titles, organisms, factors, descriptions, and experiment details. Below the search results, users can find additional filters for refining the search by organisms, factors, assay types, and project type. Search datasets are highlighted red (4), the number of items per page is highlighted (5), the Sort by options are set to release date and are highlighted red (5) and the General search Filter table is highlighted red (6). "><figcaption><p><em>The Open Science Data Repository search interface allows users to search, filter, and view search results, including titles, organisms, factors, descriptions, and experiment details. Below the search results, users can find additional filters for refining the search by organisms, factors, assay types, and project type. Search datasets are highlighted red (4), the number of items per page is highlighted (5), the Sort by options are set to release date and are highlighted red (5) and the General search Filter table is highlighted red (6).</em> </p></figcaption></figure>

<figure><img src=".gitbook/assets/Slide6.png" alt="The input text describes two sections of a search interface. Section A, titled &#x22;General Search Filters,&#x22; provides options for filtering data sources (GeneLab, ALSDA, NIH GEO, EBI PRIDE, and ANL MG-RAST) and data types (Study, Experiment, Subject, Biospecimen, and Payload). It also includes a &#x22;Show more&#x22; button. Section B, titled &#x22;Study Search Filters,&#x22; offers project-type options for filtering studies, including Ground, Spaceflight, and High Altitude. The GeneLab, ASLSDA and Study data options have been selected and the boxes next to them are orange" width="176"><figcaption><p><em>Section A, titled "General Search Filters," provides options for filtering data sources (GeneLab, ALSDA, NIH GEO, EBI PRIDE, and ANL MG-RAST) and data types (Study, Experiment, Subject, Biospecimen, and Payload). It also includes a "Show more" button. Section B, titled "Study Search Filters," offers project-type options for filtering studies, including Ground, Spaceflight, and High Altitude. The GeneLab, ASLSDA and Study data options have been selected and the boxes next to them are orange.</em></p></figcaption></figure>

1. **“General Search Filters”**
   1. Data Source
   2. Data Type
2. **“Study Search Filters”**
   1. Project Type
   2. Assay type
   3. Organism
   4. Tissue
   5. Factor

#### Metadata User Interface <a href="#i4jh5wczhr02" id="i4jh5wczhr02"></a>

**There is a lot of information available through the OSDR so it is important to read the description of each experiment carefully.**

Study metadata are viewable through a tabbed user interface for each study (see below). Click on each tab to navigate through the different sections: Description, Protocols, Sample and Assay Tables, Publications, Study Files and for some processed studies transcriptional visualization.

* The “Description” tab provides an overview of the study including a text description, contact information, experimental factors, organism(s), type of assay(s), project or mission details, funding information and citation information.
  * **Description example:** The red box highlights the “Description tab” view that has selected an example dataset, OSD-37. Below is a series of buttons that allow the user to navigate through the data to find details of interest.

<figure><img src=".gitbook/assets/Slide22 (1).png" alt=""><figcaption><p><em>The left-hand side of the webpage contains shortcut buttons that take you to additional information, such as a brief description of the experiment, the submission date of the experiment, the size of the data, the GeneLab ID associated with the data, the Digital Object Identifier (DOI), information about the source of the samples, and the assays, publications, files, and version history. The description box has been highlighted with a red dashed line. All the options on the menu on the left have been highlighted with red letters (A-K).</em></p></figcaption></figure>

#### Single study data visualization <a href="#id-882uty7gnw36" id="id-882uty7gnw36"></a>

1. Go to a study you are interested in within the OSDR or use [this link](https://osdr.nasa.gov/bio/repo/data/studies/OSD-37) to observe OSD-37.

**Note:** When examining the Uniform Resource Locator (URL) in your web browser, one can swiftly navigate between various studies by simply modifying the numeric value after the web link. This capability enables the user to swiftly transition between distinct studies.

[https://osdr.nasa.gov/bio/repo/data/studies/OSD-37](https://osdr.nasa.gov/bio/repo/data/studies/OSD-37)

[https://osdr.nasa.gov/bio/repo/data/studies/OSD-###](https://osdr.nasa.gov/bio/repo/data/studies/OSD-37)

1. At the top of the page, you should be able to see an icon picture summarizing the model organism next to the study ID number and data release version #.
2. On the left of the screen, you’ll see a series of buttons that can quickly navigate you down to different sections.
   1. Description - An abstract summary of the experiment
   2. Experiment(s) - “No associated Experiments” is not uncommon
   3. Payload(s) - “No associated Payloads” is not uncommon
   4. Mission(s) - “No associated Mission” is not uncommon
   5. Protocol(s) - provides the names and detailed descriptions of the sample collection, library construction, assay, treatment and any other protocol(s).
   6. Samples - Table of metadata describing the sample's names including quantitive and qualitative metadata values describing how they were treated. Users can select which fields they wish to export and export them as .csv files for use as factors in new subsequent models.
   7. Assay(s) - Table of metadata linking the samples to quantitive and qualitative metadata about how they were processed. Users can select which fields they wish to export as .csv files that can be used as factors in downstream statistical models.
   8. Publication(s) - Title, Authors, PubMedID, DOI## and web link to papers associated with this study. These provide a first-hand account of the study from the researchers who conducted the primary study.
   9. File(s) - A directory showing the OSD archive data folders that contain the study metadata, the raw or processed data provided by the research team and any extra processed data created by the GeneLab team.
   10. Version History - Date of the release of the current version and any associated files that were updated.
   11. Visualization - Launches into a new tab in the browser and provides insights into the GeneLab studies with processed data. “No processed data” is not uncommon.
3. Scrolling down will allow you to a description of the study, the factor(s), organism(s), assay(s), and descriptions of the related project metadata.
4. There are a lot of different types of data that can be downloaded and used for further analysis and publication.

#### Selecting sample(s) data for export <a href="#fjmk1199na87" id="fjmk1199na87"></a>

**The samples tab contains data that can be downloaded for new statistical analysis.**

*   The “Samples” and “Assays” tabs provide sample and assay level details formatted in a navigatable table. These tabs include specific organism characteristics, study factors and treatments, sample and sample processing metadata, assay execution parameters, and data processing metadata. Use the bottom navigation bar to scroll through all the columns, and the right margin scroll bar to navigate the rows.

    * **Sample example:** A OSD dataset, OSD-37, sample name tab and sort/filter functions. The select export column button in the top left corner allows the selection and download of the most important sample factors. Entries in the table with Blue text provide links to ontology databases to help define their meaning. In the bottom right corner the number of entries on the page is set to 25 as default, many studies have more samples, so you can either navigate through the samples with the arrow button(s) or increase the number of entries per page to 50 or >\~100.

    <figure><img src=".gitbook/assets/Slide23 (1).png" alt="The image shows the “Samples” tab selection which includes the name of the samples. There are scroll bars on the right side and the bottom. The table contains multiple fields of data describing the characteristics of the samples and providing links to some ontology terms that can be used to define experimental factors. The Select Export Columns button has been highlighted with a red box and a red dotted arrow."><figcaption><p><em>The image shows the “Samples” tab selection which includes the name of the samples. The table contains data describing the characteristics of the samples and providing links to some ontology terms that can be used to define experimental factors. The Select Export Columns button has been highlighted with a red box and a red dotted arrow.</em></p></figcaption></figure>

#### Selecting assay data for export <a href="#glrbshh7awz9" id="glrbshh7awz9"></a>

**The Assay(s) tab contains data that can be saved or used to develop new statistical tests.**

* There are many fields of data, users can scroll left and right using the slider bar at the bottom of the table or by “holding shift” while scrolling with a “mouse wheel”.
  * **Assays example:** A GeneLab dataset, OSD-37, assay name tab and sort/filter functions. Some studies have more than one assay type which can be accessed using the drop-down menu in the top right-hand corner. These columns can be selected for download using the export column button. Some fields are links to data files that can be downed directly from the webpage, “right-click” on the entries with blue text and select “download” from the options.

<figure><img src=".gitbook/assets/Slide25.png" alt="The image shows the “Assays” tab selection which includes the name of the assay (which in some cases is a drop-down menu that presents the option of multi-assay that can be selected). There are scroll bars on the right side and the bottom. There is a table that contains multiple fields of data describing the characteristics of the assays and providing links to some of the data products. The scroll bar at the bottom of the page is highlighted with a red box and a yellow arrow shows where the grey bar can be moved to scroll the meta-data stable to the left or right. The “Select Export Columns” button has been highlighted with a red box and an orange dotted line."><figcaption><p>Please remember that you change between different types of assay and can scroll lef and right to observe more metadata information. </p></figcaption></figure>

#### Exporting tables to csv, txt, or .xls <a href="#f2351776owgq" id="f2351776owgq"></a>

**Assay data tables can be exported from OSDR and saved or used on local or cloud-based computers.**

* Users can download selected columns from the Samples and/or Assays table. Click on the Select Export Columns button. Select the desired columns and click Export CSV or desired format.
  * **Assay example:** The OSDR dataset, OSD-37, the “select export columns” has been selected. The Export CSV can be seen at the center top of the table. The user can select which columns or “Fields” they’d like to download by selecting boxes next to the options. The button left corner has “select all”, “unselect all” and close buttons.

#### The File(s) tab provides raw and processed data <a href="#m4rj3iwtu9j" id="m4rj3iwtu9j"></a>

**This resource allows you to access and download the raw and processed data.**

* The Study Files tab provides metadata and raw or processed study data files. Each row includes information about the size, type, and description. Click on the file name to download. You can download multiple files by clicking the checkbox at the left of the file name to select and then clicking the “Download Selected Files” button. To select all files in a resource category, navigate to the desired folder by selecting the folder name and then click on the checkbox at the top of the table, next to the Files column name.
  * **File example:** Study files view and sort/filter functions for OSDR dataset, OSD-37.

<figure><img src=".gitbook/assets/Slide27.png" alt="The files tab has a “Search Files” entry field, then shows the folder structure of the selected OSD study and any subsequent folders which always include the Study Metadata Files, usually include the Raw Data files and occasionally include processed data files or GeneLab Processed data files."><figcaption></figcaption></figure>

### Search Bar <a href="#iu61uhqypy1m" id="iu61uhqypy1m"></a>

**The search bar is a quick and easy way to filter through the data and can be used to create precise database queries.**



**OSDR provides users with a full-text search capability of the metadata for all datasets.**

Full-text search terms can be a single word or multiple words with either Boolean, wildcards (asterisks \*), or string (text).

* Searches are case-insensitive concerning search term(s). To perform a search, enter a keyword or set of keywords, in the search box and either press ‘Enter’ or use your mouse to select the magnifying glass icon.
  * **Search bar example:** Search results can be sorted by relevance, release date, source, and title in ascending or descending orders. In addition, to see an overview of the study metadata, click on the image of a magnifying glass, (highlighted with a red dashed square) to show highlighted keyword search relevance.
  * Clicking on a study title will show all the detailed metadata for that study and provide further data and links or use the page navigator arrow(s) on the top right corner to go to the next page of search results.

<figure><img src=".gitbook/assets/Slide9.png" alt="The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page, the “Search Datasets” box has been highlighted."><figcaption><p><em>The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page, the “Search Datasets” box has been highlighted.</em></p></figcaption></figure>

#### Single-word search <a href="#f4y457hcuqrc" id="f4y457hcuqrc"></a>

**Below are some examples of searches.**

* **Single-term search:** These are quick and easy and usually create long lists of loosely connected studies.
  * **Single-term example:** Search results from searching the single term ‘genome’

#### Multi-word search <a href="#u3geep5fcd5d" id="u3geep5fcd5d"></a>

**Choosing two or more keywords can quickly identify studies related to your area of interest.**

* Search results will contain all the search terms and in this example, filter down to 14 OSDR accessions.
  * **Duel-word example:** Search results from searching on multiple terms ‘genome ecotype’

<figure><img src=".gitbook/assets/Slide13 (2).png" alt="The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the term “genome ecotype” has been entered. This search identifies 14 studies that use these terms in their metadata."><figcaption><p>The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the term “genome ecotype” has been entered. This search identifies 14 studies that use these terms in their metadata.</p></figcaption></figure>

#### &#x20;<a href="#yo5jgk217hqz" id="yo5jgk217hqz"></a>

#### Multiple-field searches <a href="#yo5jgk217hqz" id="yo5jgk217hqz"></a>

**How to create efficient searches by searching multiple fields.**

**Multiple-term search:** Search in multiple files such as general search filters, study search filters and keywords or phrases can be used to create concise lists of related studies.

* **Multiple-term search example:** Search results from multiple-term search combined with an additional selection of “Spaceflight” specific “Project type” are shown below.



<figure><img src=".gitbook/assets/Slide15.png" alt="The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the term “genome ecotype” has been entered. In addition, the “Space flight” has been selected as a factor and is highlighted with a red box. This search identifies 4 studies that use these terms in their metadata."><figcaption><p><em>The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the term “genome ecotype” has been entered. In addition, the “Space flight” has been selected as a factor and is highlighted with a red box. This search identifies 4 studies that use these terms in their metadata but only the first 3 are shown.</em></p></figcaption></figure>



### Boolean Operator Options <a href="#l33m3u35t1a7" id="l33m3u35t1a7"></a>

**How to make your search more insightful or precise using defined boolean operators.**

Please note that you may not use both the Boolean operators and double quotations together.

The resulting set is the same as searching for the terms ‘genome’ and ‘ecotype without the operator.

<table data-header-hidden><thead><tr><th width="190"></th><th></th></tr></thead><tbody><tr><td>Operator</td><td></td></tr><tr><td>AND</td><td>ALL search terms must be present (default Boolean search)</td></tr><tr><td>OR</td><td>ANY of your search terms can be present</td></tr><tr><td>NOT</td><td>Exclude words from your search</td></tr></tbody></table>

**Double Quotation marks define search phrases as essential for a precise match requirement.**

* If multiple words are in double quotations, then those words must match exactly in the order given, as shown below:
  * **Quotation example:** Search results from searching the exact term “genome ecotype” by using quotation marks. In this example, no search results were due to a combination of keywords and Boolean operators that failed to match any sample in the database.&#x20;

<figure><img src=".gitbook/assets/Slide14 (1).png" alt="The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the term “genome ecotype” has been entered, this time in quotation marks. This search resulted in no matches being found.  "><figcaption><p><em>The left-hand side of the webpage contains the "genotype ecotype" search results. In the center of the page the “Search Datasets” box has been highlighted, and the term “genome ecotype” has been entered, this time in quotation marks. This search resulted in no matches being found.</em> </p></figcaption></figure>

#### Boolean NOT operator. <a href="#w4aeta4y960m" id="w4aeta4y960m"></a>

**Refining search techniques to identify related studies.**

* Exclude studies containing the term(s) from your search using the minus prefix (-). This is the same as using the NOT operator which is the same as using the NOT operator.
  * **Not operator example:** Below is a search using the ‘NOT’ operator. genome ecotype –genotype.

<figure><img src=".gitbook/assets/Slide16 (1).png" alt="The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the terms “genome ecotype - genotypes” are highlighted to show how Not operators can be used to adjust the search results."><figcaption><p><em>The left-hand side of the webpage contains the "genoe ecotype - genotypes" search results. In the center of the page the “Search Datasets” box has been highlighted, and the terms “genome ecotype - genotypes” are highlighted to show how Not operators can be used to adjust the search results.</em></p></figcaption></figure>

#### Boolean AND operator. <a href="#c7u6se40fjeh" id="c7u6se40fjeh"></a>

**How to use AND operators to make your search more precise.**

* Require term(s) in your search using the plus prefix (+). This is the same as using the AND operator.
  * **An operator example:** genome sequencing +WS ecotype

<figure><img src=".gitbook/assets/Slide17 (1).png" alt="The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the terms “genome ecotype + WS ecotype” are highlighted to show how AND operators can be used to adjust the search results."><figcaption><p><em>The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the terms “genome ecotype + WS ecotype” are highlighted to show how AND operators can be used to adjust the search results.</em></p></figcaption></figure>

#### Boolean \* wild card operator. <a href="#v28y5ignyqya" id="v28y5ignyqya"></a>

**How to use a \* wild card to find closely related studies.**

* Search on unspecified portions of the search terms using an asterisk (\*)
  * **A \*wild card Boolean operator example:** genome ecotype \* Gravity. In this example, we identified 2 studies with samples that received variable quantities of gravity as a study factor.



<figure><img src=".gitbook/assets/Slide18 (1).png" alt="The NASA logo, website navigation menu, and page title are displayed in the header section. The left-hand side of the webpage contains the filters mentioned previously. In the center of the page the “Search Datasets” box has been highlighted, and the terms “genome ecotype + * Gravity” is highlighted to show how unspecified “wild card” operators can be used to adjust the search result."><figcaption><p><em>The left-hand side of the webpage contains filters with a Booealn wild card operator being used to define the gravity level. In the center of the page the “Search Datasets” box has been highlighted, and the terms “genome ecotype + * Gravity” is highlighted to show how unspecified “wild card” operators can be used to adjust the search result.</em></p></figcaption></figure>

### &#x20;<a href="#alw6ircy431n" id="alw6ircy431n"></a>

### Federating data sources <a href="#id-86tmuz33uzle" id="id-86tmuz33uzle"></a>

**How to add an external database to your search.**

OSDA has integrated, commonly termed a data federation, with multiple heterogeneous external databases. Users can search across multiple databases in addition to OSDA. The links in the federated search results are to the authoritative external databases.

OSDA is currently federated with:

* [The National Institutes of Health (NIH) Gene Expression Omnibus (GEO)](https://www.ncbi.nlm.nih.gov/geo/)
* [The European Bioinformatics Institute (EBI) Proteomics Identification (PRIDE)](https://www.ebi.ac.uk/pride/archive/)
* [The Argonne National Laboratory (ANL) Metagenomics Rapid Annotations using Subsystems Technology (MG-RAST)](http://metagenomics.anl.gov/)

The OSDA repository does not contain copies of the data sets found in the external databases, GeneLab maintains metadata records (e.g., information about the data) of the external data sets in federated databases. These records are automatically updated to keep the GeneLab database search content up to date with the external databases.

To search in one or all of these databases, enter text search term(s) and select the desired databases as shown below. Federated search results are then shown. You may change the database selection(s) at any time and the search results will be updated accordingly.

* In addition to searching the GeneLab Data Repository, federated data repositories can be included in the search.
  * **Example:** Below are examples of federated data repositories that can be searched.

<figure><img src=".gitbook/assets/Slide19.png" alt="The chart shows the user which of the specific databases have been selected to be included in the general search filters. If the user does not have a specific database in mind GeneLab and ALSDA are automatically selected, these options can be left on or turned off or they can also add NIH GEO, EBI PRIDE and AML MG-RAST to search." width="563"><figcaption><p><em>The user can select specific databases to be included in the general search filters. If the user does not have a specific database in mind GeneLab and ALSDA are automatically selected, these options can be left on or turned off or they can also add NIH GEO, EBI PRIDE and AML MG-RAST to search.</em></p></figcaption></figure>

### Study Search Filters <a href="#id-5dnlxovdz6up" id="id-5dnlxovdz6up"></a>

**The filter box on the left can filter through the OSDR to find studies related to your expertise or interests.**

In addition, OSDR offers filters that facilitate the search process for related studies within the GeneLab repository. These filters encompass Project Type, Factors, Organisms, and Assay Types. The menu associated with each category contains pre-populated terms derived from datasets included in the OSDR repository. It is noteworthy that the use of these filters is possible without the inclusion of any additional search terms.&#x20;

* After filter selection, the filter is immediately applied, potentially altering the number of search results displayed. Filter values are shown as text above the results as they are added. Multiple options from the same filter allow users to select more closely related studies.
* For instance, to filter your search, choose 'Spaceflight' from the 'Factors' menu, followed by 'RNA sequencing (RNA-seq)' and 'Seedlings' from the 'Tissue' menu. OSDR will search for studies that have either of these terms as factors.
* Users can change the filter terms they have selected by selecting the filter again from the drop-down menu or by clicking the "Clear" button. It is important to note that the "Clear" button only eliminates the filter conditions and does not erase any text search terms entered before the filters. In addition to keyword searches in the OSDA, users can also search using key factors from the metadata of the studies.
  * **Example:** Below is an example set of filters that can be used to identify studies with similar experimental designs.

<figure><img src=".gitbook/assets/Slide20.png" alt="The Image shows a series of Filters grouped by Assay type, Organism, Tissue and Factor. Spaceflight, RNAseq, Plant and Seedling were selected."><figcaption><p><em>The Image shows a series of Filters grouped by Assay type, Organism, Tissue and Factor. Spaceflight, RNAseq, Plant and Seedling were selected.</em></p></figcaption></figure>

### &#x20;<a href="#n2cdxvmjoiy1" id="n2cdxvmjoiy1"></a>

### Single Study Data Visualization <a href="#n2cdxvmjoiy1" id="n2cdxvmjoiy1"></a>

#### Launching the single-study data visualization <a href="#jirp1jnmqw4k" id="jirp1jnmqw4k"></a>

Alternatively, if you’ve identified a study of interest in the OSDA and it has processed data available then you can view it with the OSDR single study data Visualization either by launching it from the OSDR accession’s ”Visualization” button or by replacing the \* wildcards in this website URL ([https://visualization.genelab.nasa.gov/data/OSD-\*\*\*](https://visualization.genelab.nasa.gov/data/OSD-\*\*\*)) with the accession number of interest.

*   After assessing the title, study descriptions and any related research paper you can use the data visualization application to learn more about some of the quantitative patterns within the data.

    * **Single study** [**example OSD-37:**](https://visualization.genelab.nasa.gov/data/OSD-37) If you’ve already navigated to a study of interest by searching through the study metadata you can launch the visualization menu using the visualization tab on the left menu.&#x20;



<figure><img src=".gitbook/assets/Slide21.png" alt="Example showing the OSD page for accession # 37. The page presents a description of the experiment and related metadata. https://osdr.nasa.gov/bio/repo/data/studies/OSD-37."><figcaption><p><em>Example showing the OSD page for accession # 37. The page presents a description of the experiment and related metadata.</em> <a href="https://osdr.nasa.gov/bio/repo/data/studies/OSD-37">https://osdr.nasa.gov/bio/repo/data/studies/OSD-37</a>.</p></figcaption></figure>

### &#x20;<a href="#bltxtycjg7o9" id="bltxtycjg7o9"></a>

### Single Study Visualization Navigation bar <a href="#bltxtycjg7o9" id="bltxtycjg7o9"></a>

**The OSDR Visualization Navigation bar can allow you to navigate to a graph of interest.**

* After launching the OSDR visualization application on the lefthand side of the screen you’ll see a navigation window that will allow you to select to go to the "Home," "PCA," "Volcano,", “Pair plots”, “Heat maps”, “DGE”, “GSEA”, and Group Selection.
  * [**Single study navigation example**](https://visualization.genelab.nasa.gov/data/OSD-37)**:** In this example, we see Group 1 of Col-0\&FLT ( FLT is the flight group) and Group 2 is Col-0\&GC (GC is the ground control group). [https://visualization.genelab.nasa.gov/data/OSD-37](https://visualization.genelab.nasa.gov/data/OSD-37)
  * Clicking on these visualisation names allows the user to jump straight to the visualization of most interest.

<figure><img src=".gitbook/assets/Slide33.png" alt="The OSDR Visualization interface includes the text &#x22;OSDR Visualization&#x22; and &#x22;OSD-37,&#x22; and a “Study details” search bar. Below is a horizontal navigation menu with various options like &#x22;Home,&#x22; &#x22;PCA,&#x22; &#x22;Volcano,&#x22;, “Pair plots”, “Heat maps”, “DGE”, “GSEA”, and “Group Selection”. In this example, it shows that Group 1 of Col-0&#x26;FLT (which is the flight group) and Group 2 is Col-0&#x26;GC (which is the ground control group). There is a “Modify groups” button in the bottom left corner that can be used to change the groups of samples used to generate the data in the plots" width="140"><figcaption><p><em>The OSDR Visualization interface includes the text "GeneLab Visualization" and "OSD-37," and a “Study details” search bar. Below is a horizontal navigation menu with various options like "Home," "PCA," "Volcano,", “Pair plots”, “Heat maps”, “DGE”, “GSEA”, and “Group Selection”. In this example, it shows that Group 1 of Col-0&#x26;FLT (which is the flight group) and Group 2 is Col-0&#x26;GC (which is the ground control group). There is a “Modify groups” button in the bottom left corner that can be used to change the groups of samples used to generate the data in the plots</em></p></figcaption></figure>

### &#x20;<a href="#hmtojdhstykw" id="hmtojdhstykw"></a>

### Select Group(s) <a href="#hmtojdhstykw" id="hmtojdhstykw"></a>

**Select Groups can be compared statically using the GeneLab data visualisation application.**

* Space Flight (FLT) vs Ground control (GC) is a linear model often used to understand and identify genes, proteins and other omics data types that change in response to spaceflight (or related stimuli)
  * **Single study grouping example:** Select the sample group 1 and group 2 for differential comparison. You can deselect these groups by clicking on the blue check box or select them by clicking on the empty boxes. In this example, the user can compare the ecotype and treatment groups.

<figure><img src=".gitbook/assets/Slide49.png" alt="The images show the Group Selection screen. The screen is divided into two sections, each labelled with a group name and containing descriptions of the selected factors. In this example, it shows the &#x22;Ecotypes” and &#x22;treatments&#x22;. A red dashed line highlights that when 2 groups are assigned a pop-up window shows the groups that are being compared."><figcaption><p><em>The images show the Group Selection screen. The screen is divided into two sections, each labelled with a group name and containing descriptions of the selected factors. In this example, it shows the "Ecotypes” and "treatments". A red dashed line highlights that when 2 groups are assigned a pop-up window shows the groups that are being compared.</em></p></figcaption></figure>

### &#x20;<a href="#we9brgzii034" id="we9brgzii034"></a>

### Principal component analysis <a href="#we9brgzii034" id="we9brgzii034"></a>

**Principal component analysis (PCA) shows clusters of samples based on their similarity.**

* **PCA:** Click on samples to the right to select the desired samples to be viewed on the PCA plot. Icons on the upper right region of the plot allow for downloading, navigating and saving the plot. You can choose either 2D or 3D plots. If you click on the 3D graph and hold the left mouse button you can rotate the graph in 3D to optimise the angle and potentially highlight specific clustering patterns.
  * **Single study PCA example:** 3D plot has been selected and PC1, PC2, and PC3 are presented on the graph. Drop-down options allow users to replace these PC’s by selecting an alternative from the menu.
  * We can also see next to the “Color by factor” label that the current “all” is selected and the yellow arrow shows how the names of the factors have been combined in the figure legend.
  * It can be important to explore the PC's that separate the data based on different factors as the main factor such as spaceflight might not always be captured by the first or second PC's values. Sometimes results presented as a series of 2D PCA plots can be clearer than a 3D plot present 3 PC values.&#x20;

<figure><img src=".gitbook/assets/Slide34.png" alt="The scatter plot exemplifies the clustering of samples via Principal Component Analysis (PCA), a method that transforms correlated variables into uncorrelated ones called principal components. The plot presents the projection of samples onto the first two or 3 principal components, showcasing the majority of the variance in the original data. The clustering suggests the presence of distinct groups, indicating different characteristics captured by the principal components."><figcaption><p><em>The scatter plot exemplifies the clustering of samples via Principal Component Analysis (PCA), a method that transforms correlated variables into uncorrelated ones called principal components. The plot presents the projection of samples onto the first two or 3 principal components, showcasing the majority of the variance in the original data. The clustering suggests the presence of distinct groups, indicating different characteristics captured by the principal components.</em></p></figcaption></figure>

#### &#x20;<a href="#eewgwyi97mqc" id="eewgwyi97mqc"></a>

### Pair plots  <a href="#eewgwyi97mqc" id="eewgwyi97mqc"></a>

**The “Pair plots” can be used to compare 2 samples from group 1 or group 2 respectively.**

* Drop-down menus allow the researcher to select different replicates and view the comparative gene expression dispersion plots. This is useful for finding outlining samples or loci and the difference between each sample can be refined by adjusting the color threshold (default 20). In the top right-hand corner, there is a menu of graph-specific options, including snapshot, zoom, lasso, and full screen. The “update” button is highlighted in blue and needs to be pressed if you change any of the factors.
  * **Pair plot replicate comparison example:** We can see the % difference threshold is set to 20, we can also see the names of the samples being paired against one another and they are replicated of the same treatment.

<figure><img src=".gitbook/assets/Slide35 (1).png" alt="Pair plots comparing 2 samples from Atha_Col-0_s1-pool_FLT_Rep4_R1-FL-B4 on the X axis and with Atha_Col-0_s1-pool_FLT_Rep4_R1-FL-B4 on the Y-axis. Loci are represented by red and blue dots. In the top right-hand corner, there is a menu of options, including snapshot, zoom, lasso, full screen, and the “update” button is highlighted in blue."><figcaption><p><em>Pair plots comparing 2 samples from Atha_Col-0_s1-pool_FLT_Rep4_R1-FL-B4 on the X axis and with Atha_Col-0_s1-pool_FLT_Rep4_R1-FL-B4 on the Y-axis. Loci are represented by red and blue dots. In the top right-hand corner, there is a menu of options, including snapshot, zoom, lasso, full screen, and the “update” button is highlighted in blue.</em></p></figcaption></figure>

### &#x20;<a href="#id-3cb9r7if7di1" id="id-3cb9r7if7di1"></a>

### Volcano Plots <a href="#id-3cb9r7if7di1" id="id-3cb9r7if7di1"></a>

**Volcano plots show statistical significance vs magnitude of change.**

* To present the Volcano plot, the p-value is converted using a -log10 transformation to enhance the visibility of significant loci. The Fold Change (FC) is transformed using a Log2 transformation to balance the representation of upregulated and downregulated genes. This facilitates the identification of genes with significant statistical differences and substantial expression changes. Users can select parameters, samples for comparison, and the maximum adjusted p-value using the drop-down menu. The Log2 FC threshold can be adjusted using the menu bar at the top. The image can be exported or explored using the Zoom or Lasso tool to investigate specific clusters or outliers.
  * **Volcano plot example:** This graph generated by OSD-37 shows the adjusted P-value on the y-axis after it has experienced a -log10 transformation compared to the Log10 fold change (FC). In this example, _SCPL50_ appears to have a dramatic suppression during flight.

<figure><img src=".gitbook/assets/Slide36.png" alt="Image showing a volcano plot, with red dots representing genes with a significant increase in expression in orbit and blue dots representing genes that expression that was potentially suppressed by spaceflight-related factors. Drop-down menu allows for the selection of parameters, samples for comparison and maximum adjusted p-value. Some tabs also enable the transformation of the Y axis, the threshold of the adjusted P-value and the threshold of the Log2 FC can be adjusted using the menu bar at the top."><figcaption><p><em>Image showing a volcano plot, with red dots representing genes with a significant increase in expression in orbit and blue dots representing genes that expression that was potentially suppressed by spaceflight-related factors. Drop-down menu allows for the selection of parameters, samples for comparison and maximum adjusted p-value. Some tabs also enable the transformation of the Y axis, the threshold of the adjusted P-value and the threshold of the Log2 FC can be adjusted using the menu bar at the top.</em></p></figcaption></figure>

#### &#x20;<a href="#t7933tpb5wrd" id="t7933tpb5wrd"></a>

### Table of gene expression data <a href="#t7933tpb5wrd" id="t7933tpb5wrd"></a>

**A summary table of data compiling gene RefSeq IDs, Symbols, Log2Fc, P values, and Adjusted-P-values is often the most concise way to view data.**

* In the top right-hand corner, boxes allow the user to filter the table based on the maximum P-VAL (P-value), maximum ADJP (adjusted P-value), LOG2FC (Log2 transformed fold change) or to just search for gene REFSEQ ID or gene Symbol (Name). In the top left of the table, the buttons allow the user to copy, export to CVS, Excel, PDF or even directly Print.
  * **Summary of results example:** This table shows the top 10 most significantly differentially expressed loci from OSD-37. The maxim adjusted p-value is set to 0.05, in the bottom left it shows 2445 entries selected by this threshold, in the bottom right it shows that there are 245 pages of data (as the screen had been adjusted to show 10 loci per page). These can be downloaded for further analysis.

<figure><img src=".gitbook/assets/Slide38.png" alt="A screenshot from OSDR data visualization titled &#x22;OSD-37 DGE&#x22; showing a list of Gene REFSEQ IDs and Gene Symbols, along with their LOG2FC, PVAL and ADJP values. In the top right corner, there are boxes where users can enter values for P-VAL (P-value), maximum ADJP (adjusted P-value), LOG2FC (Log2 transformed fold change) or just search for particular gene REFSEQ ID or gene Symbol (Name). In the top left, there are a series of buttons to help export to CVS, Excel, PDF or Print. In the bottom left corner, it shows the number of entries on the page the total number selected by the filtering options and the total number of entries. In the bottom right corner, some buttons allow the user to navigate through the results on the screen."><figcaption><p><em>A screenshot from OSDR data visualization titled "OSD-37 DGE" showing a list of Gene REFSEQ IDs and Gene Symbols, along with their LOG2FC, PVAL and ADJP values. In the top right corner, there are boxes where users can enter values for P-VAL (P-value), maximum ADJP (adjusted P-value), LOG2FC (Log2 transformed fold change) or just search for particular gene REFSEQ ID or gene Symbol (Name).</em> <em>In the top left, there are a series of buttons to help export to CVS, Excel, PDF or Print. In the bottom left corner, it shows the number of entries on the page the total number selected by the filtering options and the total number of entries. In the bottom right corner, some buttons allow the user to navigate through the results on the screen.</em></p></figcaption></figure>

### Gene Set Enrichment Analysis (GSEA) <a href="#l6v0c891a6ps" id="l6v0c891a6ps"></a>

**GSEA is a tool that enables statistical comparisons of genetic and molecular databases.**

[“Geneset Enrichment Analysis](https://www.gsea-msigdb.org/gsea/index.jsp) (GSEA) - For a detailed explanation of GSEA statistics (ES, NES, FDR, P-value) see the [\*GSEA Statistics](https://www.gsea-msigdb.org/gsea/doc/GSEAUserGuideTEXT.htm#\_Enrichment\_Score\_\(ES\)) website. GSEA is performed using GSEApy 0.10.3, for more information about the parameters used and how to run GSEApy GSEA function: [GSEA](https://gseapy.readthedocs.io/en/latest/run.html) or [GSEApy documentation](https://gseapy.readthedocs.io/en/latest/introduction.html). All gmt files were downloaded from Enrichr: [Enrichr libraries](https://maayanlab.cloud/Enrichr/#libraries). The number of “genesets” and maximum false discovery rate (FDR) are set to their default values 6 and 0.25 respectively. The Permutations, type, min size and max size are all left as default.

* The are many plot types that be used to view these results including, Enrichment Score Plot, NES Plot, Dot Plot, Ridge Ploy and Network Plot.
  * **Example GSEA:** Before starting the analysis select the plot type you wish to use. This example uses a NES Plot (bar plot) but there are often logical reasons to present different types of data in different types of plot(s) to help disguise between results.

<figure><img src=".gitbook/assets/Slide39.png" alt="Image displaying a table of graphing options. The selection of options has created a red NES plot showing 4 terms (Ribosome, Endocytosis, RNA degradation and cell cycle) identified by GSEA analysis of the KEGG_2019 database. The permutation and gene list min and maximum parameters are highlighted with an orange dashed box. The plot-type menu has been highlighted with an orange dotted box."><figcaption><p><em>Image displaying a table of graphing options. The selection of options has created a red NES plot showing 4 terms (Ribosome, Endocytosis, RNA degradation and cell cycle) identified by GSEA analysis of the KEGG_2019 database. The permutation and gene list min and maximum parameters are highlighted with an orange dashed box. The plot-type menu has been highlighted with an orange dotted box.</em></p></figcaption></figure>

#### &#x20;<a href="#o9we3slvz8rf" id="o9we3slvz8rf"></a>

### Ontology databases and statistical methods used for analysis. <a href="#o9we3slvz8rf" id="o9we3slvz8rf"></a>

**Selecting different databases for statistical analysis can identify different responses and provide alternative perspectives and insights from the results.**

* Numerous ontology databases and statistical methodologies are available for application to the processed DESeq2 results.
  * **Ontology database example description:** A menu of pop-up menu databases can be opened and one selected based on your model organism and question of choice. 5 different statical methods can be used by GSEA, signal to noise, T test, Fold change, Difference of class and Log2 Fold change.

<figure><img src=".gitbook/assets/Slide40.png" alt="Image of half of the OSDR visualization page, the “Gene Set” menu has KEGG_2019 selected and a red dotted line links the list of accessible databases that can be selected from the connected drop-down menu that is highlighted by a yellow dotted line. The image also shows a red dashed line listing the different analysis “Methods” that can be used and the “Fold change” option is highlighted in blue to show it is selected." width="563"><figcaption><p><em>Image of half of the OSDR visualization page, the “Gene Set” menu has KEGG_2019 selected and a red dotted line links the list of accessible databases that can be selected from the connected drop-down menu that is highlighted by a yellow dotted line. The image also shows a red dashed line listing the different analysis “Methods” that can be used and the “Fold change” option is highlighted in blue to show it is selected.</em></p></figcaption></figure>

#### &#x20;<a href="#bj3385d7e61m" id="bj3385d7e61m"></a>

### Dot plots <a href="#bj3385d7e61m" id="bj3385d7e61m"></a>

**Dot plots can present 3 types of data on the same graph.**

The OSDR facilitates the visualization of the normalized enrichment score, false discovery rate, and Gene ratio values of all differentially expressed pathways on a single graph for user evaluation.

* **Phenotype Permutation vs. Gene Set Permutation:** Choosing the appropriate permutation method can influence your results. Phenotype permutation is generally preferred as it maintains the biological context of your experiment.
  * **Example dot plot:** demonstrating the analysis of the OSD-37 KEGG\_2019 pathway employing the Fold change method and “Phenotype” permutation type. The Minimum and maximum size parameters were retained at their default values of 15 and 500 loci, respectively, as per standard practice. The quantities of “geneset”s” and the maximum false discovery rate (FDR) are set to their default values of 6 and 0.25, respectively.

<figure><img src=".gitbook/assets/Slide41.png" alt="A screenshot of an Open Science Data Repository (OSDR) webpage. The page displays a Dotplot graph titled &#x22;OSD-37 GSEA.&#x22; The x-axis label is &#x22;Normalized /enrichment Score&#x22; and the y-axis shows the name of the enriched KEGG pathway. The graph used two colors, red and blue to emphasize the variation in the FDR values for this data series. Data points size displays the Gene ratio value which is connected by lines to the pathway name on the Y-axis. Red dotted lines highlight where the Dot ploy options can be found and that in this example the “Phenotype” permutation has been selected."><figcaption><p><em>A screenshot of  a Dotplot graph titled "OSD-37 GSEA." The x-axis label is "Normalized /enrichment Score" and the y-axis shows the name of the enriched KEGG pathway. The graph used two colors, red and blue to emphasize the variation in the FDR values for this data series. Data points size displays the Gene ratio value which is connected by lines to the pathway name on the Y-axis. Red dotted lines highlight where the Dot ploy options can be found and the “Phenotype” permutation has been selected.</em></p></figcaption></figure>





### Ridge plots <a href="#zc2ql3hfusf9" id="zc2ql3hfusf9"></a>

**A ridge plot combines a density plot and a histogram to show how data, like Fold Change (FC), is distributed.**

The OSDR facilitates the visualization of the false discovery rate (FDR), and Foldchange values of all differentially expressed pathways on a single graph for user evaluation. This visualization aids in determining whether genes are predominantly upregulated, downregulated, or exhibit minimal change. The smooth, ridge-like form reveals trends, and the plot is compatible with log-transformed data, a common feature in fold-change analysis. Overlaying plots for various groups allows researchers to compare fold-change distributions.

* Ridge plots are effective tools for present GSEA analysis.
  * **Example Ridge plot:** This example ridge-plot was made with a “Fold Change” of loci from the KEGG\_2019 GSEA analysis data from OSD-37. The Minimum size and maximum size values were left to the default 15 and 500 loci as standard. The number of ““geneset”s” and maximum false discovery rate (FDR) are set to their default values 6 and 0.25 respectively.

<figure><img src=".gitbook/assets/Slide42.png" alt="A screenshot of an Open Science Data Repository (OSDR) ridge plot page displays a ridge plot entitled &#x22;OSD-37 GSEA”. The Y-axis label shows the names of the terms detected by enrichment analysis of the selected gene sets eg Endocytosis and Cell cycle. The x-axis label is &#x22;Gene Fold Change”. Red dotted line highlights where the “Ridge Plot&#x22; option was engaged."><figcaption><p><em>A screenshot of an Open Science Data Repository (OSDR) ridge plot page displays a ridge plot entitled "OSD-37 GSEA”. The Y-axis label shows the names of the terms detected by enrichment analysis of the selected gene sets eg Endocytosis and Cell cycle. The x-axis label is "Gene Fold Change”. Red dotted line highlights where the “Ridge Plot" option was engaged.</em></p></figcaption></figure>

### &#x20;<a href="#v5vfnpxc0gtj" id="v5vfnpxc0gtj"></a>

### Network Plot <a href="#v5vfnpxc0gtj" id="v5vfnpxc0gtj"></a>

**A network plot uses dots that represent nodes of data points that describe the response detected in the analysis.**

* Network plots are effective tools for showing the variation in the number of genes involved in a detected response.
  * **Example network diagram:** it can be observed that the Ribosome possesses a greater number of loci that were implicated in that response ontology group. Even though "RNA degradation" and "Cell cycle" demonstrate a lesser quantity of associated genes, it is evident that they possess heightened significance due to their diminished adjusted P-value, as delineated by the gradient ranging from blue to red.

<figure><img src=".gitbook/assets/Slide43.png" alt="A screenshot of an Open Science Data Repository (OSDR) network plot page with the title OSD-37 GSEA. In the center a plot shows 4 circles, representing enriched terms from the KEGG_2019 database. The size of the dots indicates the number of genes detected in that GO group and the red-to-blue color gradient shows the significance of their enrichment. A red dotted line highlights where the network plot was selected"><figcaption><p><em>A screenshot of an Open Science Data Repository (OSDR) network plot page with the title OSD-37 GSEA. In the center a plot shows 4 circles, representing enriched terms from the KEGG_2019 database. The size of the dots indicates the number of genes detected in that GO group and the red-to-blue color gradient shows the significance of their enrichment. A red dotted line highlights where the network plot was selected</em></p></figcaption></figure>

####

### Network Plot used in conjunction with pathway illustration

**A network plot uses dots that represent nodes of data points that describe the response detected in the analysis.**

* Network plots can be merged with pathway diagrams to help illustrate connections between enriched terms and concepts like cellular function. Figures can then be shared with other OSDR AWG collaborators on platforms like GitHub as staging for eventual publication in Zenodo and peer-reviewed journals.

<figure><img src=".gitbook/assets/Slide44.png" alt="The previous network dots and plot information have been incorporated into the endocytosis diagram to illustrate how these KEGG pathway terms related to cellular biology. The use of pop-out boxes and dotted or dashed lines is used to show connections based on knowledge from the literature."><figcaption><p><em>The previous network dots and plot information have been incorporated into the endocytosis diagram to illustrate how these KEGG pathway terms related to cellular biology. The use of pop-out boxes and dotted or dashed lines is used to show connections based on knowledge from the literature.</em></p></figcaption></figure>

### &#x20;<a href="#j5wz6ra5953b" id="j5wz6ra5953b"></a>

### NES plots <a href="#j5wz6ra5953b" id="j5wz6ra5953b"></a>

**NES (Normalized Enrichment Score) plot offers a visual representation of the normalized enrichment score for gene sets identified by the user based on their parameters.**

* NES plots are an effect tool for viewing pathways that are identified by GSEA.
  * **Example NES plot:** The data from OSD-37 provides the normalized enrichment score adjacent to terms that possess enrichment in the selected database. KEGG\_2019 was used as the “geneset” for enrichment analysis. However, in this instance, we show it is possible to change the database analyzed. The bar plot then displays GO terms identified during the analysis when the GO\_cellular\_Componet\_AutoRIF gene set database was queried.

<figure><img src=".gitbook/assets/Slide45 (1).png" alt=""><figcaption><p><em>This image shows a NES plot and the options used to generate it. Situated at the top of the webpage is a search bar that facilitates the exploration of specific terms within the datasets and other “geneset" for analysis. Immediately beneath the search mechanism is a tabular structure providing alternative plot types and graph parameters. The bar plot structure summarizes distinct gene groups that are known to function in a similar location based on the GO-Cellular_Component_AutoRIF enrichment analysis. The Red dotted lines highlight the “number of “geneset”s” and the method values have been adjusted from their default values.</em></p></figcaption></figure>





### Enhancing NES plots

**Adjusting the parameters to influence the quantity and sequence of pathways detected.**

* NES plots have a numerical constant that can be adjusted to modify the appearance pathways that are identified by GSEA
  * **Numerical method example:** In this OSD-37 example we’ve changed the “Method” value to 0 (classic), which changes the normalized enrichment score and graph layout.

<figure><img src=".gitbook/assets/Slide46.png" alt="This image shows a screenshot from the OSDR Visualization software showing an NES plot graph and the options used to generate it. The bar plot structure summarizes distinct gene groups that are known to function in a similar location based on the GO-Cellular_Component_AutoRF enrichment analysis. The red dotted line highlights that the “Method” is currently set to 0 (classic mode) but was set to 1 in the previous figure."><figcaption><p><em>This image shows a NES plot and the options used to generate it. The bar plot structure summarizes distinct gene groups that are known to function in a similar location based on the GO-Cellular_Component_AutoRF enrichment analysis. The red dotted line highlights that the “Method” is currently set to 0 (classic mode) but was set to 1 in the previous figure.</em></p></figcaption></figure>



**Refining NES plots using the Method Value**

**Adjusting the “geneset” is highly recommended in the Method tab and can change the number and order of pathways identified.**

* When utilizing a normalized enrichment score plot (NES plot) to depict pathway enrichment via the fold change method, the graphical representation not only illustrates pathways with statistical significance but incorporates pathways that lack such significance.
  * **Number of “genesets” example:** In this example, the “Method” value is set to 1, and we’ve changed the “geneset” to Cellular\_Component\_AutoRIF\_Predicted\_zscore. Notice that the number of “genesets” is now set to 15 and that the color coding shows that only 4 of these are significant based on the current threshold of a maximum False Discovery Rate of 0.25.

<figure><img src=".gitbook/assets/Slide47.png" alt="This image displays a screenshot of the OSDR Visualization software, which includes an NES plot graph and the parameters used to create it. The NES bar plot structure summarizes distinct gene groups known to function in a similar cellular location based on GO-Cellular_Component_AutoRIF_zscore enrichment analysis. The method is currently set to 0 1, and the number of “geneset”s included was set to 15, with coloring determined by a 0.25 False Discovery Rate (FDR) cutoff threshold. Red arrows highlight the Gene sets and number of “geneset”s selected for this enrichment analysis."><figcaption><p><em>This image displays a screenshot of the OSDR Visualization software, which includes an NES plot graph and the parameters used to create it. The NES bar plot structure summarizes distinct gene groups known to function in a similar cellular location based on GO-Cellular_Component_AutoRIF_zscore enrichment analysis. The method is currently set to 0 1, and the number of “geneset”s included was set to 15, with coloring determined by a 0.25 False Discovery Rate (FDR) cutoff threshold. Red arrows highlight the Gene sets and number of “geneset”s selected for this enrichment analysis.</em></p></figcaption></figure>



### Fine-tuning permutations

**Adjusting the permutation value can be used to improve the GSEA statistical model.**

GSEA calculates enrichment scores by comparing the data to a random null distribution. This null distribution is generated by permuting the phenotype labels of your samples numerous times. A higher number of permutations leads to a more precise estimation of the p-value associated with your enrichment score.

* You can choose the number of permutations to run, with a higher number leading to potentially more accurate p-values.
  * **Example Finetuning Permutations:** Altering the default premonition from 300 to 500 in the example description resulted in a significant change in the FDR of the "cytoskeleton" Gene Ontology term.

<figure><img src=".gitbook/assets/Slide48.png" alt="This image displays a screenshot of the GeneLab Visualization software, which includes an NES plot graph and the parameters used to create it. The NES bar plot structure summarizes distinct gene groups known to function in a similar cellular location based on GO-Cellular_Component_AutoRIF_zscore enrichment analysis. The method is currently set to 1, and the number of “geneset”s included was set to 15, with coloring determined by a 0.25 False Discovery Rate (FDR) cutoff threshold. The permutations number has been increased to 500 and the cytoskeleton GO term has increased its NES score and changed from the Red FDR>0.25 to the Blue FDR<0.25 group. Red arrows highlight the Permutations value has been increased and that moving the mouse over a bar creates a popup that provides its name and value."><figcaption><p><em>This image displays NES plot graph and the parameters used to create it. The NES bar plot structure summarizes distinct gene groups known to function in a similar cellular location based on GO-Cellular_Component_AutoRIF_zscore enrichment analysis. The method is currently set to 1, and the number of “genesets” included was set to 15, with coloring determined by a 0.25 False Discovery Rate (FDR) cutoff threshold. The permutations number has been increased to 500 and the cytoskeleton GO term has increased its NES score and changed from the Red FDR>0.25 to the Blue FDR&#x3C;0.25 group. Red arrows highlight the permutation value has been increased and that moving the mouse over a bar creates a popup that provides its name and value.</em></p></figcaption></figure>

### &#x20;<a href="#xez5bpa6ns59" id="xez5bpa6ns59"></a>
